superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js

```diff
@@ -162,7 +176,13 @@ describe('Dashboard tabs', () => {
     });
 
     // click row level tab, send 1 more query
-    cy.get('.tab-content ul.nav.nav-tabs li').last().click();
+    cy.get('[data-test="dashboard-component-tabs"]')
+      .last()
+      .find('[data-test="nav-list"]')
+      .children()
+      .as('row-level-tabs');
```

Comment:
Appreciate all these `as(...)` changes! Much easier to read, in my book.

Reply:
Yes it is cool!